e2e-tests/functions

```diff
@@ -2,7 +2,8 @@
 
 GIT_COMMIT=$(git rev-parse HEAD)
 GIT_BRANCH=${VERSION:-$(git rev-parse --abbrev-ref HEAD | sed -e 's^/^-^g; s^[.]^-^g;' | sed -e 's/_/-/g' | tr '[:upper:]' '[:lower:]')}
-API="psmdb.percona.com/v1-13-0"
+API="psmdb.percona.com/v1"
+OPERATOR_VERSION=$(grep -oP 'Version = "\K([0-9\.]*)' ${test_dir}/../../version/version.go)
```

Comment:
Are you sure about it ? It does not work for me:
```
grep -oP 'Version = "\K([0-9\.]*)' ./e2e-tests/../../version/version.go
grep: ./e2e-tests/../../version/version.go: No such file or directory
```
Also please check why 'Build docker image' stage does not fail in case when docker image can't be built at all. We spend more than 8 hours for unneeded tests :(